variables.tf

```
@@ -73,6 +73,23 @@ variable "application_stack" {
   default = null
 }
 
+variable "virtual_applications" {
+  description = "An object of virtual application path mappings for Windows Web Apps."
```

Update description with correct variable type, and when the value of this variable is applicable:

```suggestion
  description = "A list of virtual applications to configure for this Web App. Only applicable if value of kind is \"Windows\"."
```